diff --git a/libraries/chain/CMakeLists.txt b/libraries/chain/CMakeLists.txt
index 67f51fc8b..5209cce79 100644
--- a/libraries/chain/CMakeLists.txt
+++ b/libraries/chain/CMakeLists.txt
@@ -145,6 +145,12 @@ add_library(Boost::numeric_ublas ALIAS boost_numeric_ublas)
 
 target_link_libraries( eosio_chain PUBLIC bn256 fc chainbase eosio_rapidjson Logging IR WAST WASM
                        softfloat builtins ${CHAIN_EOSVM_LIBRARIES} ${LLVM_LIBS} ${CHAIN_RT_LINKAGE}
+                       LLVMTransformUtils
+                       LLVMExecutionEngine
+                       LLVMObject
+                       LLVMInstCombine
+                       LLVMScalarOpts
+                       LLVMMC		       
                        Boost::signals2 Boost::hana Boost::property_tree Boost::multi_index Boost::asio Boost::lockfree
                        Boost::assign Boost::accumulators
                      )
